products/sle/main.pm

```diff
@@ -480,7 +480,10 @@ sub load_online_migration_tests {
         loadtest "migration/online_migration/zypper_migration";
     }
     loadtest "migration/online_migration/post_migration";
-    loadtest "console/check_system_info" if (is_sle && (get_var('FLAVOR') =~ /Migration/) && (get_var('SCC_ADDONS') !~ /ha/) && !is_sles4sap && (is_upgrade || get_var('MEDIA_UPGRADE')));
+    if (is_sle && (get_var('FLAVOR') =~ /Migration/) && (get_var('SCC_ADDONS') !~ /ha/) && !is_sles4sap && (is_upgrade || get_var('MEDIA_UPGRADE'))) {
+        loadtest "console/check_os_release";
```

Comment:
Where is the content of console/check_os_release? Why not put the code in this PR to review together?

Reply:
It already existed for a long time. Written by others. I just added this module to our migration test flow